public/components/navigation.react.js

```diff
@@ -9,12 +9,19 @@ export default class NavigationComponent extends React.Component {
     }
 
     render () {
+        var distinctContentSources = [];
+        var allContentSources = this.props.data;
+        for(var i = allContentSources.length - 1; i >= 0; i--) {
```

Comment:
Yes, it's because of the order in which the results are processed. Going from `0...length` will result in setting as default the last environment in the list.